stdlib/std.ncl

```diff
@@ -1171,36 +1176,33 @@
     id
       : forall a. a -> a
       | doc m%"
-        The identity function, that is
-
-        ```nickel
-        id x = x
-        ```
-
-        for any value `x`.
-      "%
-      = fun x => x,
-
-    const
-      : forall a b. a -> b -> a
-      | doc m%"
-        Always returns the first argument, ignoring the second.
+        The identity function, that is, `id x` is `x` for any value `x`.
 
         # Examples
 
         ```nickel
-        const 5 42 == 5
+        std.function.id null
+          => null
+        (std.function.id (fun x => x + 1)) 0
+          => 1
         ```
       "%
-      = fun x y => x,
+      = fun x => x,
 
     compose
       : forall a b c. (b -> c) -> (a -> b) -> (a -> c)
       | doc m%"
-        Function composition, from right to left. That is,
+        Function composition, from right to left. That is, `compose f g x` is
+        `f (g x)`.
+
+        # Example
```

Comment:
```suggestion
        # Examples
```
We're using the plural everywhere else, if I remember right. Even if there is only a single example given.